src/components/TokenLocking/Leaderboard.tsx

```diff
@@ -46,71 +50,121 @@ const StyledTableRow = styled(TableRow)(({ theme }) => ({
   },
 }))
 
-const StyledTable = styled(Table)(({ theme }) => ({
-  borderCollapse: 'separate',
-  '& tr:first-child td': {
+const HighlightedTableRow = styled(TableRow)(({ theme }) => ({
+  // hide last border
+  '& td:first-child': {
+    borderRadius: '6px 0 0 6px',
+  },
+  '& td:last-child': {
+    borderRadius: '0 6px 6px 0',
+  },
+  '& td': {
     backgroundColor: theme.palette.background.light,
     background:
       'linear-gradient(var(--mui-palette-background-light), var(--mui-palette-background-light)) padding-box,linear-gradient(to bottom, #5FDDFF 12.5%, #12FF80 88.07%) border-box',
-    border: '1px solid transparent',
+    border: '1px solid transparent !important',
   },
-  '& tr:first-child td:before': {},
-  'tr:first-child td:not(:first-child)': {
-    borderLeft: 'none',
+  '& td:not(:first-child)': {
+    borderLeft: 'none !important',
   },
-  '& tr:first-child td:not(:last-child)': {
-    borderRight: 'none',
+  '& td:not(:last-child)': {
+    borderRight: 'none !important',
   },
+
+  '& td:before': {},
 }))
 
+const StyledTable = styled(Table)(({ theme }) => ({
+  borderCollapse: 'separate',
+}))
+
+const LookupAddress = ({ address }: { address: string }) => {
+  const name = useEnsLookup(address)
+
+  return (
+    <>
+      <Box display="flex" flexDirection="row" gap={2} alignItems="center">
+        <Identicon size={32} address={address}></Identicon>
+        {name ? name : address}
+      </Box>
+    </>
+  )
+}
+
+const OwnRank = () => {
+  const ownRankResult = useOwnRank()
+  const { data: ownRank } = ownRankResult
+
+  if (ownRank) {
+    return (
+      <HighlightedTableRow key={ownRank.holder}>
+        <StyledTableCell align="center">
+          {ownRank.position <= 3 ? (
+            <SvgIcon
+              component={
+                ownRank.position === 1 ? FirstPlaceIcon : ownRank.position === 2 ? SecondPlaceIcon : ThirdPlaceIcon
+              }
```

Comment:
nit: We have this logic in multiple places. Could also be an enum or an object to access the correct icon based on the position.